packages/jsii-pacmak/test/generated-code/__snapshots__/target-java.test.ts.snap

```diff
@@ -5626,7 +5626,7 @@ package software.amazon.jsii.tests.calculator;
  * Calculator calculator = new Calculator();
  * calculator.add(5);
  * calculator.mul(3);
- * System.out.println(calculator.getExpression().getValue());
+ * System.out.println(calculator.expression.getValue());
```

Comment:
Same here.

The new code is probably correct, but why did this change all of a sudden?